moto/core/responses.py

```diff
@@ -221,6 +278,22 @@ def _get_param(self, param_name, if_none=None):
         val = self.querystring.get(param_name)
         if val is not None:
             return val[0]
+
+        # try to get json body parameter
+        try:
+            j = json.loads(self.body)
+            # raise key error if key really does not exist
+            return j[param_name]
+        except:
```

Comment:
This `except` catches a bit too much. How about we add clauses for `except JSONDecodeError` and `except KeyError` for the two things that could go wrong here?

```
try:
  return json.loads(self.body)[param_name]
except JSONDecodeError: pass
except KeyError: pass
```

Reply:
thanks, I will use Value error instead of JSONDecodeError to support both python2 and python3

Reply:
And, made new line for each exception handling because linter warns.